src/main/java/com/faforever/client/game/RatingChangedLabelController.java

```diff
@@ -0,0 +1,56 @@
+package com.faforever.client.game;
+
+import com.faforever.client.fx.Controller;
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.replay.Replay.PlayerStats;
+import com.faforever.client.util.RatingUtil;
+import javafx.css.PseudoClass;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class RatingChangedLabelController implements Controller<Node> {
+  private static final PseudoClass POSITIVE = PseudoClass.getPseudoClass("positive");
+  private static final PseudoClass NEGATIVE = PseudoClass.getPseudoClass("negative");
+  public Label ratingChangedLabelRoot;
+  private final I18n i18n;
+
+  @Inject
+  public RatingChangedLabelController(I18n i18n) {
+    this.i18n = i18n;
+  }
+
+  @Override
+  public Node getRoot() {
+    return ratingChangedLabelRoot;
+  }
+
+  @Override
+  public void initialize() {
+
+  }
+
+  public void setRatingChange(PlayerStats playerStats) {
+    if (playerStats.getAfterMean() == null || playerStats.getAfterDeviation() == null) {
+      return;
+    }
+    int newRating = RatingUtil.getRating(playerStats.getAfterMean(), playerStats.getAfterDeviation());
+    int oldRating = RatingUtil.getRating(playerStats.getBeforeMean(), playerStats.getBeforeDeviation());
+
+    String changedRating = String.valueOf(newRating - oldRating);
```

Comment:
String.valueOf() should not be used to represent numbers in the UI as it's not locale-sensitive. Keep `changedRating` an `int`. I'd also rename it to `ratingChange`. `changedRating` sounds more like a Rating object.